### PR TITLE
Update assessments format when serializing.

### PR DESCRIPTION
**Pull request by symbolist**

### CODE REVIEW

**Comment by symbolist** [comment]
@muzaffaryousaf Can you give this a quick look?

**Comment by waheedahmed** [comment]
lgtm :+1:

File: openassessment/xblock/xml.py [hunk 1]
```diff
@@ -760,7 +761,7 @@ def serialize_examples_to_xml_str(assessment):
         A unicode string of the XML serialized examples.
 
     """
-    examples = assessment.get('examples', [])
+    examples = update_assessments_format([assessment])[0].get('examples', [])
```
Comment (muzaffaryousaf) [inline openassessment/xblock/xml.py]: @symbolist i can't understand why we are choosing 0th element. can you please explain?

**Comment by muzaffaryousaf** [comment]
other than one small question.  :+1:

File: openassessment/xblock/xml.py (hunk 1 above)
Comment (symbolist) [inline openassessment/xblock/xml.py]: `update_assessments_format` takes a list of assessments so we put the `assessment` in a list, and after the list has been updated we retrieve the `assessment` with `[0]`.

**Pull request merged**